src/nm-manager.c

```diff
@@ -5641,6 +5665,12 @@ get_property (GObject *object, guint prop_id,
 	case PROP_VERSION:
 		g_value_set_string (value, VERSION);
 		break;
+	case PROP_CAPABILITIES:
+		g_value_set_variant (value, g_variant_new_fixed_array (G_VARIANT_TYPE ("u"),
+															   priv->capabilities,
+															   G_N_ELEMENTS (priv->capabilities),
+															   sizeof(priv->capabilities[0])));
```

Comment:
Or use this:

https://cgit.freedesktop.org/NetworkManager/NetworkManager/tree/contrib/editors/networkmanager-style.el?h=nm-1-4